src/metabase/pulse/task/send_pulses.clj

```diff
@@ -153,24 +153,26 @@
 
 (declare update-send-pulse-trigger-if-needed!)
 
+(defn- active-dashsub-pcs
+  []
+  (t2/select :model/PulseChannel
+             {:select    [:pc.*]
+              :from      [[:pulse_channel :pc]]
+              :left-join [[:pulse :p] [:= :pc.pulse_id :p.id]
+                          [:report_dashboard :d] [:= :p.dashboard_id :d.id]]
+              :where     [:and
+                          [:= :pc.enabled true]
+                          ;; only do this for dashboard subscriptions, alert has been
+                          ;; migrated to notifications
+                          [:not= :p.dashboard_id nil]
+                          [:= :d.archived false]]}))
+
 (defn init-dashboard-subscription-triggers!
   "Update send pulse triggers for all active pulses.
   Called once when Metabase starts up to create triggers for all existing PulseChannels"
   []
   (assert (task/scheduler) "Scheduler must be started before initializing SendPulse triggers")
-  (task/delete-all-triggers-of-job! send-pulse-job-key)
```

Comment:
This was the line I missed the first time I review this.

Reply:
oh that explains it!